### PR TITLE
CS-3144 add uri when xmlns:c uri is not set.

### PR DESCRIPTION
**Pull request by jeremlb**
@ContentSquare/application

### CODE REVIEW

**Review by tleporcher — APPROVED** [review]

File: dom.js [hunk 1]
```diff
@@ -1004,7 +1004,7 @@ function serializeToString(node,buf,isHTML,nodeFilter,visibleNamespaces){
 			var attr = attrs.item(i);
 			if (needNamespaceDefine(attr,isHTML, visibleNamespaces)) {
 				var prefix = attr.prefix||'';
-				var uri = attr.namespaceURI;
+				var uri = (!!node.namespaceURI) ? node.namespaceURI : './lite';
```
Comment (JMDidon) [inline dom.js]: Shouldn't it be `attr.namespaceURI` here?

File: package.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
 {
   "name": "xmldom",
-  "version": "0.1.27",
+  "version": "0.1.28",
```
Comment (JMDidon) [inline package.json:3]: A little question: is a PR to their repo planned?

File: dom.js [hunk 3]
```diff
@@ -1004,7 +1004,7 @@ function serializeToString(node,buf,isHTML,nodeFilter,visibleNamespaces){
...
 				var ns = prefix ? ' xmlns:' + prefix : " xmlns";
 				buf.push(ns, '="' , uri , '"');
 				visibleNamespaces.push({ prefix: prefix, namespace:uri });
```
Comment (tleporcher) [inline dom.js]: Space after `:`

File: dom.js [hunk 4]
```diff
@@ -1014,7 +1014,7 @@ function serializeToString(node,buf,isHTML,nodeFilter,visibleNamespaces){
 		// add namespace for current node
 		if (needNamespaceDefine(node,isHTML, visibleNamespaces)) {
 			var prefix = node.prefix||'';
```
Comment (tleporcher) [inline dom.js]: space after and before `||`

**Review by tleporcher — APPROVED** [review]

**Review by Eriflow — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Eriflow) [reply package.json:3]: I think not because @jeremlb told me they refused a PR from @lseguin42 so we forked the project.

File: dom.js (hunk 4 above)
Comment (Eriflow) [reply dom.js]: @poupougnac wat ?

Comment (pmaoui) [reply dom.js]: sorry nvm

File: dom.js (hunk 1 above)
Comment (jeremlb) [reply dom.js]: nope, an attribute doesn't have any namespaceURI.

File: package.json (hunk 2 above)
Comment (jeremlb) [reply package.json:3]: exactly

Comment (jeremlb) [inline package.json]: exactly

**Pull request merged**